src/material-experimental/mdc-form-field/directives/error.ts

```diff
@@ -22,11 +22,15 @@ export const MAT_ERROR = new InjectionToken<MatError>('MatError');
   selector: 'mat-error',
   host: {
     'class': 'mat-mdc-form-field-error mat-mdc-form-field-bottom-align',
-    'role': 'alert',
+    '[attr.aria-live]': 'ariaLive',
```

Comment:
Please add `aria-atomic="true"`. Without this, if an error message changes from "Value must have only lowercase letters" to "Value must have 3-5 characters", a SR may wrongly read just "3-5 characters".

Reply:
done